.github/workflows/clang-format.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A workflow to check clang format
+name: clang format check
+
+on:
+  pull_request_target:
```

Comment:
just saw you are not getting correct merged share with `pull_request_target + pre-commit/action`.
then try using pull_request directly

Reply:
Yes. Seems it always checkout the default branch in the tests. Will fix this. Thanks!